public/javascripts/SVValidate/src/modal/ModalMission.js

```diff
@@ -92,6 +94,17 @@ function ModalMission (uiModalMission, user) {
         uiModalMission.closeButton.on('click', _handleButtonClick);
     }
 
+    //detect if mobile
+    function isMobile() {
```

Comment:
can't this function be put into a shared place, like a utils file that everyone has access to?

Reply:
I made a new js file just for this function, I don't know if you'll like that. I figured to put it in a greater scope so we could also use it on other pages in the website(landing page). I'll look to find a link for it.